lib/mix/tasks/phx.digest.clean.ex

```diff
@@ -21,14 +21,14 @@ defmodule Mix.Tasks.Phx.Digest.Clean do
   ## Options
 
     * `-o, --output` - indicates the path to your compiled
-      assets directory. Defaults to `priv/static`.
+      assets directory. Defaults to `priv/static`
```

Comment:
The documentation had a mix of list items ending with and without a dot. I decided to follow Elixir's style and remove dots at the end :)